org.andrew.ytdownloader.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.andrew.ytdownloader
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: ytdownloader
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/ytdownloader/flatpak-node/cache
+        npm_config_cache: /run/build/ytdownloader/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      # Install npm dependencies
+      - npm install --offline
+      # Build the app; in this example the `dist` script
+      # in package.json runs electron-builder
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked /app/main
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ../run.sh
+    subdir: main
+    sources:
+      - type: repo
+        url: https://github.com/aandrew-me/ytdownloader-flatpak.git
+        dest: main
+      - generated-sources.json
```

Comment:
this file is missing.

Reply:
How would I include it if it's in the git repo?

Reply:
It is not in THIS git repository. See also my question as why you don't use the original repository.

Reply:
It is in this repository. In my original repo I am using ffmpeg-static, while in this one I am using ffmpeg-static-electron. Also my original repo's package.json contains many build properties which are not needed here.